src/screens/Drawer/Calendar/CalendarScreen.tsx

```diff
@@ -27,6 +27,11 @@ export const CalendarScreen = (props: CalendarScreenProps) => {
         (state) => state.calendarReducer.previewPaneVisible,
     );
 
+    useEffect(() => {
+        //checks for notification permission, requests expo token, and updates db with updated expo token
+        useCheckPermissions();
```

Comment:
technically it's not really a hook is it? Hooks inside another hook isn't recommended anyways. This looks more like an api call

Reply:
yeah its really just a function, I'll pbly rename it once i find a good place for it to live

Reply:
just stuff it in the services folder? it's kinda weird that you have a hooks folder, and this function in it isn't a hook

Reply:
i'm going to restructure the services/shared folder in another PR so i'll leave it here for now